.git-commit-template

```diff
@@ -0,0 +1,9 @@
+COMPONENT: Subject
+
+Explanation
+
+Resolves:
+https://fedorahosted.org/freeipa/ticket/XXXX
+
+# Try to keep the subject line within 52 chars ----|
+# Also please try to not exceed 72 characters of length for the body --|
```

Comment:
IMO it would be better if these comments were directly above the subject and body, respectively.